pack/td.json

```diff
@@ -489,6 +617,80 @@
         "uniqueness": false
     },
     {
+        "code": "13035",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 3,
+        "flavor": "\"Strictly speaking, it isn't illegal to use an interrogation simulator as the blueprint for ICE.\" - Mason Bellamy",
+        "illustrator": "Andreas Zafiratos",
+        "keywords": "Code Gate - AP - Tracer",
+        "pack_code": "td",
+        "position": 35,
+        "quantity": 3,
+        "side_code": "corp",
+        "strength": 1,
+        "text": "[subroutine] <trace>trace 4</trace> Do 1 brain damage.",
```

Comment:
a very small thing I've noticed in the live site, the _**Trace**_ should be capitalised.